### DISCUSSION TITLE
How to trigger screenshots for bookmarks made before hoarder took screenshots?

### DISCUSSION THREAD

**Discussion by ovizii** [karakeep-app/karakeep#196] [Q&A] [answered]
my older bookmarks have no screenshots:

![image](https://github.com/hoarder-app/hoarder/assets/837902/b8c11ab0-fe23-4d26-bf23-e92f2aec321f)

The admin interface shows no manual trigger for screenshots:

![image](https://github.com/hoarder-app/hoarder/assets/837902/9f6a53f8-2412-40ae-b572-d383a4d53db2)

**Comment by kamtschatka** [comment, marked as answer]
you can either recrawl all links or you go to the bookmark without the screenshot and press "refresh"

**Reply by ovizii** [reply]
Sorry, I had tried refresh before, but apparently I didn't wait long enough for the screenshot to actually be generated :-)
